docs/bosh/README.md

```diff
@@ -398,13 +451,41 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Use `vim` or `nano` to create a certificate named `ca_cert.pem`:
+
+    ```
```

Comment:
Updated to include the cert in the manifest's `/misc/ca_cert`.

I validated this is fine by deploying a director using the updated manifest.